wms_svc_profile/locust_eng175719.py

```diff
@@ -0,0 +1,129 @@
+        self.syndication_prof_params = syndication_prof_params
+        self.attr_search_params = attr_search_params
+        self.search_association_params = search_association_params
+        self.profile_uuid = syndication_prof_uuid
+        self.auth = auth
+        self.headers = headers
+        self.mock = Faker()
+
+    @task()
+    def get_v2_insec_comp_ag(self):
+        try:
+            resp = self.client.get(url=self.v2_insec_comp_ag_endpoint,
+                                   params=self.insec_comp_ag_params)
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_v2_insec_office_ag(self):
+        try:
+            resp = self.client.get(url=self.v2_insec_office_ag_endpoint,
+                                   params=self.insec_comp_ag_params)
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_v2_comp_ag(self):
+        try:
+            resp = self.client.get(url=self.v2_comp_ag_endpoint,
+                                   params={**self.insec_comp_ag_params, **self.auth})
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_v2_office_ag(self):
+        try:
+            resp = self.client.get(url=self.v2_office_ag_endpoint,
+                                   params={**self.insec_comp_ag_params, **self.auth})
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_public_key(self):
+        try:
+            resp = self.client.get(url=self.public_key_endpoint,
+                                   params=self.auth)
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_profile_v2(self):
+        try:
+            resp = self.client.get(url=f"{self.v2_profile_endpoint}{self.profile_uuid}",
+                                   params=self.auth)
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_profile_v2_syndication(self):
+        try:
+            resp = self.client.get(url=f"{self.syndication_profile_endpoint}{self.profile_uuid}",
+                                   params=self.auth)
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    @task()
+    def get_profile_v2_syndication_search(self):
+        try:
+            resp = self.client.get(url=self.syndication_profile_endpoint,
+                                   params={**self.syndication_prof_params, **self.auth})
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    # @task()
```

Comment:
endpoint times out, leaving commented for now.

Reply:
Does this endpoint still time out?

Reply:
yes, it is still consistently timing out